README.md

```diff
@@ -190,6 +238,29 @@ To update the realm configuration on the locally running Keycloak instances:
 5. Refresh the locally running Keycloak pages to see the changes.
 6. Repeat steps 3 to 5 as necessary.
 
+### Updating Emails
+
+There are two components to consider when making changes to emails that Keycloak sends to users:
+* the GovUK Notify Spi contained in the govuk-notify-spi directory
+* the GovUK Notify template contained within the GovUK Notify service
+
+Parameters from Keycloak to the GovUK template are passed using the personalisation Map in the NotifyEmailSenderProvider 
+
+The key in the personalisation Map corresponds to the name of the personalisation variable defined in the template, for example: `((keycloakMessage))`
+
+1. Make the necessary changes locally and build and run the Keycloak docker image locally
+2. Make any necessary changes to the GovUKNotify TDR Intg service
+ * this can be done by making changes to the existing keycloak template, or by creating a new test template and passing the test template's id in the docker `run` command environment variable: `GOVUK_NOTIFY_TEMPLATE_ID`
+3. Deploy the new changes to Intg environment
+4. Before deploying to each environment ensure any GovUK Notify template changes are made to the template in GovUK Notify service for the TDR environment:
+ * TDR Intg
+ * TDR Staging
+ * TDR
```

Comment:
This reminds me - did you check the name of the service with Kirsten? I'd have thought it would need to be "Transfer Digital Records" rather than using the acronym.

Reply:
She has just confirmed "Transfer Digital Records"